buildbot/osuosl/master/config/builders.py

```diff
@@ -2651,6 +2651,29 @@
                         "-DCOMPILER_RT_BUILD_SANITIZERS=OFF",
                         "-DLLVM_CCACHE_BUILD=ON"])},
 
+    {'name' : "flang-arm64-windows-msvc-testsuite",
+    'tags'  : ["flang"],
+    'workernames' : ["linaro-armv8-windows-msvc-06"],
+    'builddir': "flang-arm64-win-msvc-ts",
+    'factory' : ClangBuilder.getClangCMakeBuildFactory(
+                    vs="manual",
```

Comment:
Manual means what exactly?

Is that where buildbot is started in an environment where the Visual Studio environment is already applied?
```
vcvarsall.bat
buildbot-agent
```
Like that? Instead of buildbot running vcvars for each build.